SQLBuilder/Universal/Expr/BinaryExpr.php

```diff
@@ -20,7 +20,11 @@ public function __construct($operand, $op, $operand2) {
     }
 
     public function toSql(BaseDriver $driver, ArgumentArray $args) {
-        return $this->operand . ' ' . $this->op . ' ' . $driver->deflate($this->operand2, $args);
+        $column = $this->operand;
+        if ($driver->quoteColumn) {
```

Comment:
We should not assume operand1 to be always column name, it's possible to have an expression like `true == true`. So,  a column type check will be required here.

It's best to have a ColumnName class that could be deflated into SQL string with the specific driver.

That way, we only need to extend the deflate method from Driver, instead of changing all expression classes.